minifi-toolkit/minifi-toolkit-configuration/src/main/java/org/apache/nifi/minifi/toolkit/configuration/ConfigMain.java

```diff
@@ -251,6 +268,10 @@ public int transform(String[] args) {
                     System.out.println();
                     printTransformUsage();
                     return ERR_UNABLE_TO_TRANFORM_TEMPLATE;
```

Comment:
realize this is already existing, but perhaps we should adjust to correct spelling error while in this code area